index.js

```diff
@@ -96,8 +118,58 @@ const ScrollableTabView = React.createClass({
     }
   },
 
+  updateSceneKeys(children,sceneKeys=[],currentPage){
+    let newKeys=[]
+    children.forEach((child,idx) => {
+      let key=this._makeSceneKey(child,idx)
+      if(this._keyExists(sceneKeys,key) || currentPage == idx) newKeys.push(key)
+    })
+    return newKeys;
+  },
+
+  _keyExists(sceneKeys,key){
+    return sceneKeys.findIndex((sceneKey)=> key == sceneKey) > -1
```

Comment:
We just check if its existing, so not much off a difference.
I made changes though.